scanapi/scan.py

```diff
@@ -64,3 +68,8 @@ def write_report(results):
     """
     reporter = Reporter(settings["output_path"], settings["template"])
     reporter.write(results)
+
+
+def write_without_generating_report(results):
```

Comment:
@JRubics can we also have the docstring here? Thanks!!

Reply:
done :)